frontends/web/src/locales/en/app.json

```diff
@@ -1296,6 +1318,7 @@
     "addAccount": "Add account",
     "bitboxBase": "BitBoxBase",
     "bitboxBaseConnect": "Connect BitBoxBase",
+    "buy": "Buy crypto",
     "device": "Manage device",
     "exchanges": "Buy & sell",
```

Comment:
No longer needed or change to something else? Seems confusing to have `Buy crypto` above and `Buy & sell`.

Reply:
I'll remove it here, but importantly `sidebar.exchanges` will have to be deleted in locize when we merge this into master.